index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Toolbar -->
+    <nav class="toolbar">
+        <button type="button"><img src="logo/Icon - Menu.png" alt=""></b>
+    </nav>
```

Comment:
[OPTIONAL] 
- [ ] It could be better for accessibility if you added a value to the `alt` attribute, kindly do so.